ramls/item.json

```diff
@@ -57,7 +57,18 @@
       "properties": {
         "callNumber": {
           "type": "string",
-          "description": "Effective Call Number is an identifier assigned to an item or its holding and associated with the item."
+          "description": "Effective Call Number is an identifier assigned to an item or its holding and associated with the item.",
+          "readonly": true
+        },
+        "prefix": {
```

Comment:
These new properties need reflecting in the interface versions (in both the descriptor and RAML files)